fonts/fonts.css

```diff
@@ -0,0 +1,26 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: 
+      url('Montserrat-Bold.woff2') format('woff2'),
+      url('Montserrat-Bold.woff') format('woff');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LOCO__Oxana/pullRequest?prid=5565108)